### PR TITLE
Update autocomplete pattern

### PR DESCRIPTION
**Pull request by bgeuken**
Adopts our pattern library to recent changes made in PR 6671.

https://github.com/openSUSE/open-build-service/pull/6671

### CODE REVIEW

**Comment by hennevogel** [comment]
Deploy preview for *obs-patterns* ready!

Built with commit 6cf620179f921b048c66129eb4f53de9504573f6

https://deploy-preview-51--obs-patterns.netlify.com

File: _posts/molecules/2018-10-01-autocomplete.html [hunk 1]
```diff
@@ -3,35 +3,21 @@
 title:  Autocomplete
 category: molecules
 description: Whenever a field has the ability to autocomplete its value, we should add a hint in form of placeholder and 
-             also an icon on the left that shows a spinner while loading from remote
+             also an icon on the left that shows a spinner while loading from remote.
+             OBS provides an autocomplete parti, including javascript for handling the autocomplete request and spinner. For
```
Comment (dmarcoux) [inline _posts/molecules/2018-10-01-autocomplete.html]: autocomplete parti? What is a parti?

Comment (dmarcoux) [reply _posts/molecules/2018-10-01-autocomplete.html]: We should capitalize `Javascript`.

File: _posts/molecules/2018-10-01-autocomplete.html [hunk 2]
```diff
@@ -3,35 +3,21 @@
...
+             further customization of autocomplete events please refer to the jquery autocomplete documentation (https://jqueryui.com/autocomplete/).
```
Comment (dmarcoux) [inline _posts/molecules/2018-10-01-autocomplete.html]: It should be `jQuery` and the link should be `[jQuery autocomplete documentation](https://jqueryui.com/autocomplete/)`

File: _posts/molecules/2018-10-01-autocomplete.html [hunk 3]
```diff
@@ -3,35 +3,21 @@
...
+// Below is the content of the autocomplete partial. Render it in your view, e.g.
+// "render partial: 'webui/autocomplete', locals: { html_id: 'linked_project', label: 'Original project name', source: autocomplete_projects_path }"
+.form-group.ui-front
+  %label{ for: 'input-id' }
+    Label:
+  .input-group
+    .input-group-prepend
+      %span.input-group-text
+        %i.fas.fa-search
+    %input.form-control#input-id{ class: 'obs-autocomplete form-control', data: { source: '/path/to/autocomplete/data' },
```
Comment (dmarcoux) [inline _posts/molecules/2018-10-01-autocomplete.html]: The classes should be added directly (so not inside `{}`). `form-control` is also added twice.

**Review by dmarcoux — CHANGES_REQUESTED** [review]

File: _posts/molecules/2018-10-01-autocomplete.html (hunk 1 above)
Comment (bgeuken) [reply _posts/molecules/2018-10-01-autocomplete.html]: Actually it's `JavaScript`

File: _posts/molecules/2018-10-01-autocomplete.html [hunk 4]
```diff
@@ -3,35 +3,21 @@
 title:  Autocomplete
 category: molecules
 description: Whenever a field has the ability to autocomplete its value, we should add a hint in form of placeholder and 
-             also an icon on the left that shows a spinner while loading from remote
+             also an icon on the left that shows a spinner while loading from remote.
+             OBS provides an autocomplete partial, including JavaScript for handling the autocomplete request and spinner. For
+             further customization of autocomplete events please refer to the [jQuery autocomplete documentation](https://jqueryui.com/autocomplete/).
```
Comment (dmarcoux) [inline _posts/molecules/2018-10-01-autocomplete.html]:
```suggestion
             further customization of autocomplete events, please refer to the [jQuery autocomplete documentation](https://jqueryui.com/autocomplete/).
```

File: _posts/molecules/2018-10-01-autocomplete.html [hunk 5]
```diff
@@ -3,35 +3,21 @@
...
+// Below is the content of the autocomplete partial. Render it in your view, e.g.
+// "render partial: 'webui/autocomplete', locals: { html_id: 'linked_project', label: 'Original project name', source: autocomplete_projects_path }"
+.form-group.ui-front
+  %label{ for: 'input-id' }
+    Label:
+  .input-group
+    .input-group-prepend
+      %span.input-group-text
+        %i.fas.fa-search
+    %input.form-control.obs-autocomplete.form-control#input-id{ data: { source: '/path/to/autocomplete/data' },
```
Comment (dmarcoux) [inline _posts/molecules/2018-10-01-autocomplete.html]:
```suggestion
    %input.obs-autocomplete.form-control#input-id{ data: { source: '/path/to/autocomplete/data' },
```

File: _posts/molecules/2018-10-01-autocomplete.html (hunk 3 above)
Comment (dmarcoux) [reply _posts/molecules/2018-10-01-autocomplete.html]: Still valid, I added a suggestion.

**Comment by bgeuken** [comment]
@dmarcoux updated

**Comment by dmarcoux** [comment]
I added some suggestions yesterday. Please have a look. After that, it's all good.

**Comment by bgeuken** [comment]
@dmarcoux Overlooked them before. Please have another look

**Review by dmarcoux — APPROVED** [review]

**Pull request merged**